### ISSUE TITLE
ActiveStorage::AnalyzeJob fails if blob has been destroyed

### ISSUE THREAD

**Issue by ball-hayden** [rails/rails#37613]
### Steps to reproduce

1) Define a model with an attached file
2) Create a new record with an attachment
3) Destroy the record before `ActiveStorage::AnalyzeJob` has run

### Expected behavior

The job should succeed, perhaps logging that it did nothing because the blob doesn't exist anymore.

### Actual behavior

The job fails with the following error:

```
Error while trying to deserialize arguments: Couldn't find ActiveStorage::Blob with 'id'=<blah>
```

This, in itself, isn't too much of an issue. However, backends (such as Sidekiq) are expected to retry failed jobs, so the queued job is likely to fail many times with no hope of ever succeeding.

Again, in isolation, this isn't too much of an issue. In our use case we regularly expect attachments to 
be destroyed fairly quickly after creation as they pass through a chain of processing jobs. This has ended up in 100s of `ActiveStorage::AnalyzeJob`s building up and being retried repeatedly.

### System configuration
**Rails version**: Rails 6.0.0

**Ruby version**: Ruby 2.6.3

**Comment by georgeclaghorn** [comment]
Fixed in 06f8baf.

**Comment by ball-hayden** [comment]
🏎💨 
Wow. Thanks @georgeclaghorn
